src/widget/OktaSignIn.js

```diff
@@ -117,9 +116,21 @@ var OktaSignIn = (function () {
     require.config(requireConfig);
     OktaAuth = require('vendor/OktaAuth');
     Util = require('util/Util');
+    LoginRouter = require('LoginRouter');
+
 
     authClient = new OktaAuth({ uri: options.baseUrl, transformErrorXHR: Util.transformErrorXHR });
-    _.extend(this, getProperties(authClient, options));
+    _.extend(this, LoginRouter.prototype.Events, getProperties(authClient, options, LoginRouter));
+
+    // Triggers the event up the chain so it is available to the consumers of the widget.
+    this.listenTo(LoginRouter.prototype, 'all', function() {
```

Comment:
Same nit here about replacing with:
```javascript
this.listenTo(LoginRouter.prototype, 'all', this.trigger);
```
I realize that it's the way it was in framework/view - not sure why they did that.